### PR TITLE
update resources.json all remaing keys

### PR DESCRIPTION
**Pull request by rnrnshn**
updated the values of keys that exist in both json files on the new repo and the old repo

### CODE REVIEW

File: data/resources.json [hunk 1]
```diff
@@ -214,24 +207,20 @@
...
     "formTitle": "Get the latest updates, insights and news about GDG DevFest Ukraine sent to your inbox.",
     "emailRequired": "Please enter a valid email address.",
     "subscribe": "Subscribe",
     "subscribed": "Subscribed",
-    "close": "Close",
+    "close": "Fechado",
     "generalError": "Some error has occured. Please, try again later.",
     "toast": "Successfully subscribed!"
   },
-  "team": {
```
Comment (osvaldoM) [inline data/resources.json:229]: you were not supposed to remove the existing keys.. I'll take the liberty to update your PR.

Comment (rnrnshn) [reply data/resources.json:229]: Oh my bad... 

**Old key:value**
``` "team": "Organizadores" ``` 

**New key:value**

`"team": {
    "description": "Google is known all around the world. Everyone is 'googling', checking on 'maps' and communicating in 'gmail'. For simple users, they are services that just works, but not for us. Developers see much more: APIs, scalability issues, complex technology stacks. And that is what GDG is about.<br><br> [Google Developers Group (GDG) Lviv](http://lviv.gdg.org.ua/) - is open and volunteer geek community who create exciting projects and share experience about Google technologies with a passion.<br><br> Our goal is to organize space to connect the best industry experts with Ukrainian audience to boost development of IT."
  },`

So I thought it needed to change but I ended up removing it

File: data/resources.json [hunk 2]
```diff
@@ -179,11 +176,7 @@
       "link": "/blog/"
     }
   },
-  "speakers": {
-    "skills": "Skills",
-    "previousYears": "Years",
-    "previousYear": "Year"
-  },
+   "speakers": "Palestrantes",
```
Comment (osvaldoM) [inline data/resources.json]: just undo this change and I'll approve your PR

File: data/resources.json [hunk 3]
```diff
@@ -23,14 +23,11 @@
   "country": "Country",
   "time": "Time",
   "faq": "/data/faq.md",
-  "cod": "/data/cod.md",
+  "cod": "Codigo de conduta",
```
Comment (osvaldoM) [inline data/resources.json]: keep this as /data/cod.md

**Comment by osvaldoM** [comment]
@0limpi0 , please check this one.

**Comment by rnrnshn** [comment]
I'm working on it

**Review by osvaldoM — APPROVED** [review]
looks mergeble to me

**Pull request merged**